docs/interactionmanager.md

```diff
@@ -37,6 +37,156 @@ By default, queued tasks are executed together in a loop in one `setImmediate` b
 
 ---
 
+## Example
+
+### Basic
+
+```SnackPlayer name=InteractionManager%20Function%20Component%20Basic%20Example&supportedPlatforms=ios,android
+import React, { useState, useEffect } from "react";
+import {
+  Alert,
+  Animated,
+  InteractionManager,
+  Platform,
+  StyleSheet,
+  Text,
+  View,
+} from "react-native";
+
+const instructions = Platform.select({
+  ios: "Press Cmd+R to reload,\n" + "Cmd+D or shake for dev menu",
+  android:
+    "Double tap R on your keyboard to reload,\n" +
+    "Shake or press menu button for dev menu",
+});
+
+const useMount = func => useEffect(() => func(), []);
+
+const useFadeIn = (duration = 5000) => {
+  const [opacity] = useState(new Animated.Value(0));
+
+  // Running the animation when the component is mounted
+  useMount(() => {
+    // Animated.timing() create a interaction handle by default, if you want to disabled that
+    // behaviour you can set isInteraction to false to disabled that.
+    Animated.timing(opacity, {
+      toValue: 1,
+      duration,
+    }).start();
+  });
+
+  return opacity;
+};
+
+const Ball = ({ onShown }) => {
+  const opacity = useFadeIn();
+
+  // Running a method after the animation
+  useMount(() => {
+    InteractionManager.runAfterInteractions(() => onShown());
+  });
+
+  return <Animated.View style={[styles.ball, { opacity }]} />;
+};
+
+const App = () => {
+  const [show, setShow] = React.useState(false);
```

Comment:
One more static call left. Other than that LGTM. 

Thank you so much for your contribution and collaboration!